src/components/pip/pip.scss

```diff
@@ -1,24 +1,23 @@
 @import '~styles/variables';
 @import '~styles/styles';
 
+$animation-duration: 0.3s;
+
 .childPlayer {
   position: absolute;
 
-  .videoContainer {
-    border-radius: 8px;
-    overflow: hidden;
-    box-shadow: 0 0 40px 0 rgba(0, 0, 0, 0.2), 0 0 4px 0 rgba(0, 0, 0, 0.3);
+  &:not(.dragging) {
+    transition: all $animation-duration ease-in;
   }
 
-  &.bottom {
-    bottom: 0;
+  &.playerHover {
+    margin-top: 50px;
```

Comment:
this should be 37px ( calculated by vars of button height + button gap)

Reply:
there is an issue with interactive area, let's discuss how and where we gonna fix current bug